kyo-prelude/shared/src/main/scala/kyo2/kernel/Effect.scala

```diff
@@ -13,10 +13,7 @@ object Effect:
         new KyoDefer[A, S]:
             def frame = summon[Frame]
             def apply(v: Unit, context: Context)(using Safepoint) =
```

Comment:
Can you remind me why defer needs `Unit`?

Reply:
All effects need an input value to suspend with. In case of `Defer`, it's a `Unit` placeholder since the effect only provides suspension and doesn't use its input.